CH8527 | Update The Pennylane Demos Landing Page (#328)
PennyLaneAI/qml: demonstrations.rst

=====

.. meta::

>    :property="og:description": Take a deeper dive into quantum computing by exploring cutting-edge algorithms using PennyLane and near-term quantum hardware.

Reviewer: ```suggestion
   :property="og:description": Take a deeper dive into quantum computing by exploring cutting-edge algorithms using PennyLane and quantum hardware.
```

    <div class="container">
        <!-- Section: Features v.1 -->
            <p class="lead grey-text text-center mx-auto mb-5">

>             Take a deeper dive into quantum computing by exploring cutting-edge algorithms using PennyLane and near-term quantum hardware.</p>

Reviewer: ```suggestion
            Take a deeper dive into quantum computing by exploring cutting-edge algorithms using PennyLane and quantum hardware.</p>
```

                                    <br>
                                    <strong>QML</strong>
                                    </h3>
                                    <p class="mb-1 grey-text px-3">Explore cutting-edge research in quantum machine learning using PennyLane.</p>
                                    <div class="white-text d-flex justify-content-center mt-auto"><h5>Read more <i class="fas fa-angle-double-right"></i></h5></div>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
            </div>

            <div class="row justify-content-center main-cards">
                <div class="col-lg-3 mb-4">
                    <a href="demos_quantum-chemistry.html">
                        <div class="card rounded-lg h-100">
                            <div class="text-center d-flex align-items-center pb-2 h-100" style="justify-content: center;">
                                <div class="d-flex flex-column h-100">
                                    <h3 class="card-title">

>                                     <img src="_static/quantum-chemistry.png" class="img-fluid" style="max-width: 110px;"></img>

Reviewer 2: Are the card heights still dependent on the value of `max-width` here?

Author: Not the height of the cards, but the alignment of the content of the cards. The images aren't all the same size, so the `max-width` is being used to make the images look more consistent 🙂

Reviewer 2: Ah, got it!

                                    <br>
                                    <strong>Quantum Chemistry</strong>
                                    </h3>

>                                     <p class="mb-1 grey-text px-3">Investigate, simulate, and infer properties of molecules and materials using quantum computing.</p>

Reviewer: ```suggestion
                                    <p class="mb-1 grey-text px-3">Study the properties of molecules and materials using quantum computing.</p>
```

                                    <div class="white-text d-flex justify-content-center mt-auto"><h5>Read more <i class="fas fa-angle-double-right"></i></h5></div>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-lg-3 mb-4">
                    <a href="demos_quantum-computing.html">
                        <div class="card rounded-lg h-100">
                            <div class="text-center d-flex align-items-center pb-2 h-100" style="justify-content: center;">
                                <div class="d-flex flex-column h-100">
                                    <h3 class="card-title">
                                    <img src="_static/quantum-computing.png" class="img-fluid" style="max-width: 66px;"></img>
                                    <br>
                                    <strong>Quantum Computing</strong>
                                    </h3>

>                                     <p class="mb-1 grey-text px-3">Explore general quantum computing tasks and algorithms, from quantum volume to Boson sampling.</p>

Reviewer: ```suggestion
                                    <p class="mb-1 grey-text px-3">Explore general quantum computing tasks and algorithms, from quantum volume to boson sampling.</p>
```

                                    <div class="white-text d-flex justify-content-center mt-auto"><h5>Read more <i class="fas fa-angle-double-right"></i></h5></div>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-lg-3 mb-4">
                    <a href="demos_community.html">
                        <div class="card rounded-lg h-100">
                            <div class="text-center d-flex align-items-center pb-2 h-100" style="justify-content: center;">
                                <div class="d-flex flex-column h-100">
                                    <h3 class="card-title">

>                                     <img src="_static/community.png" class="img-fluid" style="max-width: 130px;"></img>

Reviewer 2: Might be good to reduce the size of this image, it looks quite large now compared to the others 😆

Author: Does this work for you?
<img width="954" alt="Screen Shot 2021-09-17 at 3 18 50 PM" src="https://user-images.githubusercontent.com/8475900/133842368-ce0d4651-3775-4ecb-8e61-0ba843c1937b.png">

Reviewer 2: Looks great, thanks @AndrewGardhouse!
